corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -299,8 +299,16 @@
   }
 }
 
-.question-tile-row .gr-no-children.gr-has-no-questions.gr-has-no-nested-questions {
-  // Prevents a priority issue that's happened three times where
-  // a group with hidden values creates whitespace.
+.gr-has-no-nested-questions {
   display: none;
 }
+
+.checkbox, .radio {
+  input[type="checkbox"], input[type="radio"] {
+    // This needs customization any time the font size in Web Apps is updated
+    margin-top: 3.15px;
```

Comment:
I wonder if [vertical-align](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align) could help here. Specifically `text-top`

Reply:
Does it have any warnings the `vertical-align` in the browser dev tools? If I remember correctly when I was playing around with it, it did not like to be combined with some display options.